src/features/game/events/landExpansion/plant.ts

```diff
@@ -408,6 +361,32 @@ export function getPlotYieldAmount({
     amount *= 1.1;
   }
 
+  if (
+    isCollectibleBuilt({ name: "Scarecrow", game }) ||
+    isCollectibleBuilt({ name: "Kuebiko", game })
+  ) {
+    amount *= 1.2;
+  }
+
+  if (inventory.Coder?.gte(1)) {
+    amount *= 1.2;
+  }
+
+  //Bumpkin Skill boost Green Thumb Skill
+  if (skills["Green Thumb"]) {
+    amount *= 1.05;
+  }
+
+  //Bumpkin Skill boost Master Farmer Skill
+  if (skills["Master Farmer"]) {
+    amount *= 1.1;
+  }
+
+  // Additive boosts below
```

Comment:
Consider: In my scripts I have a 3 line comment above the `mul` group and another 3 line comment above the `add` group.  Makes it easier to not mess up when adding new ones.  For me at least.